docs/advanced/catkin_shell_verbs.rst

```diff
@@ -0,0 +1,36 @@
+Shell support in ``catkin`` command
+===================================
+
+When you source the following file, you can use bash/zsh shell functions which provide added utility.
+
+.. code-block:: shell
+
+    . /opt/ros/indigo/etc/bash_completion.d/catkin_shell_verbs.{bash,zsh}
```

Comment:
I'm not sure what the right thing to do here is, maybe @jbohren can advise?

You could do something cute like this:

```
$ source `catkin --locate-extra-shell-verbs`
```

And then install the shell scripts that need to be sourced in with the Python code and provide an option to expose their final location.